lib/pure/collections/tables.nim

```diff
@@ -2360,7 +2360,7 @@ proc inc*[A](t: var CountTable[A], key: A, val: Positive = 1) =
   var index = rawGet(t, key)
   if index >= 0:
     inc(t.data[index].val, val)
-    if t.data[index].val == 0: dec(t.counter)
+    if t.data[index].val == 0: dec(t.counter) # how could this happen?
```

Comment:
I think this could have happened before when `val` was not `Positive`.

Reply:
updated code to `assert t.data[index].val != 0` ; if/when CountTable starts supporting dec and 0 counts, we'll revisit this